loop_ivr/management/commands/product_team_mailer.py

```diff
@@ -0,0 +1,229 @@
+        no_sms_users = get_active_user_comp_info(today_caller_object_sms_user_count, yesterday_caller_object_sms_user_count, '')
+
+        print 'SMS users %s'%no_sms_users
+
+        # Correct Queries
+        today_caller_object_correct_query_sms_count = today_caller_object.filter(call_source=3, info_status=1).count()
+        yesterday_caller_object_correct_query_sms_count = yesterday_caller_object.filter(call_source=3, info_status=1).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        per_correct_code_entered_sms = get_active_user_comp_info(today_caller_object_correct_query_sms_count, yesterday_caller_object_correct_query_sms_count, '')
+
+        # Incoming Calls
+        today_caller_object_call_count = today_caller_object.filter(call_source__in=[1, 2]).count()
+        yesterday_caller_object_call_count = yesterday_caller_object.filter(call_source__in=[1, 2]).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        no_incoming_call = get_active_user_comp_info(today_caller_object_call_count, yesterday_caller_object_call_count, '')
+
+        today_caller_object_correct_query_call_count = today_caller_object.filter(call_source__in=[1, 2], info_status=1).count()
+        yesterday_caller_object_correct_query_call_count = yesterday_caller_object.filter(call_source__in=[1, 2], info_status=1).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        per_correct_code_entered_call = get_active_user_comp_info(today_caller_object_correct_query_call_count, yesterday_caller_object_correct_query_call_count, '')
+
+        # Wrong Code Entered via call
+        today_caller_object_wrong_query_count = today_caller_object.filter(info_status=2).count()
+        yesterday_caller_object_wrong_query_count = yesterday_caller_object.filter(info_status=2).count()
+
+        comparison_param_label_str = comparison_param_label[str(options['days'])]
+        no_wrong_query_code = get_active_user_comp_info(today_caller_object_wrong_query_count, yesterday_caller_object_wrong_query_count, '')
+
+        # Total SMS Sent using textlocal
+        today_smsstatus_obj = SmsStatus.objects.filter(price_info_incoming=today_caller_object)
+        no_sms_sent = today_smsstatus_obj.count()
+        no_sms_dilivered = today_smsstatus_obj.filter(status='D').count()
+
+        # Queries with rates available
+        today_rates_available_count = today_caller_object.filter(is_rate_available__in=[2, 3]).count()
+
+        # values to be calculated
+        no_call_backs_time_limit = ''
+        no_first_attempt_success = today_caller_object.filter(info_status=1, prev_query_code__isnull=False).count()
+        
+        df = pd.DataFrame(list(SmsStatus.objects.filter(price_info_incoming=today_caller_object).values('id','price_info_incoming','status', 'delivery_time', 'api_call_initiation_time')))
```

Comment:
similary here, today_caller_object is a list

Reply:
It's not a list